sample_file_pr.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+import sys
+import boto3
+import botocore
+import argparse
+
+session = None
+
+
+class initArgs(object):
+
+    def __init__(self):
+        self.parser = argparse.ArgumentParser()
+        self.parser.add_argument(
+            "--no-verify-ssl", action='store_true', help="Disable SSL verification.")
+        self.parser.add_argument(
+            "--profile", type=str, help="The credential profile used to execute this job.  This configuration is not required if 'aws configure' has been run and a [default] profile is defined in ~/aws/crednetial", required=False)
+        self.args = self.parser.parse_args(sys.argv[1:])
+
+    def get_args(self):
+        return self.args
+
+
+def getRegions():
+    try:
+        client = session.client('ec2', verify=args.no_verify_ssl)
+    except botocore.exceptions.NoRegionError as e:
+        print("Error initiating AWS client!")
+        print("Have you configured your AWS region?  (e.g. in ~/aws/crednetial)  See 'aws configure' ")
+        print("If you are using a specific profile, use --profile")
+        exit(1)
+
+    regions = []
+
+    try:
+        for region in client.describe_regions()['Regions']:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To create a `list`, try to use `list` comprehension instead of a loop. List comprehension is the preferred way to make a list using Python, and it&#39;s simpler and easier to understand than using a loop.

[Learn more](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions)